_test/bootstrap.sh

```diff
@@ -2,12 +2,6 @@
 # This installs a the prerequisites for a given GHC/Cabal config
 set -x
 set -e
-if [ ! -z "$GHCVER" ]; then
```

Comment:
I will try to completely rewrite `bootstrap.sh` to use `dependencies` if I get the time. In #162 I removed it but, if we want to keep tests with cabal, we need to at least to centralize the data in `dependencies.txt`.